packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.Props.ts

```diff
@@ -177,6 +182,11 @@ export interface IGroupRenderProps {
    * Override which allows the caller to provider a customer footer.
    */
   onRenderFooter?: IRenderFunction<IGroupDividerProps>;
+
+  /**
+   * Flag to indicate whether to ignore the collapsing icon on header.
+   */
+  collapseAllVisibility?: CollapseAllVisibility;
```

Comment:
If you don't set the prop it is already default visible, but I've made it explicit.